test/test_datasource.py

```diff
@@ -1,6 +1,8 @@
 import unittest
+import sys
 from io import BytesIO
 import os
+sys.path.append('d:\\dev\\git\\server-client-python\\.eggs\\requests_mock-1.9.1-py3.7.egg')
```

Comment:
Is this necessary for this test to run properly?

Reply:
Nope!  Meant to remove that, thanks for catching.